lib/shopify_api/session.rb

```diff
@@ -6,10 +6,9 @@ class ValidationException < StandardError
   end
 
   class Session
-    cattr_accessor :api_key
-    cattr_accessor :secret
-    cattr_accessor :protocol
+    cattr_accessor :api_key, :secret, :protocol, :myshopify_domain, :port
     self.protocol = 'https'
+    self.myshopify_domain = 'myshopify.com'
```

Comment:
I think shopify_app could just use this, since it is already dependant on the shopify_api gem.